### ISSUE TITLE
Negate search on custom client attribute "environment" doesn't work

### ISSUE THREAD

**Issue by pietervogelaar** [sensu/uchiwa#726]
I would like to search for all servers that are not production.

According to the documentation (https://docs.uchiwa.io/guides/search-queries/) this should be:

```
environment:^((?!production).)*$
```

But nothing is found. Am I doing something wrong?

**Comment by palourde** [comment]
@pietervogelaar Are you using a custom attribute named `environment`? If so, you need to make sure every client has this attribute. 

Otherwise, if you are referring to the actual Uchiwa datacenters, you should use something like `dc:^((?! production).)*$`.

**Comment by pietervogelaar** [comment]
@palourde Yes, it's a custom attribute and every client has the attribute. But it still doesn't work.
